### PR TITLE
update input binding code block - ReadMe

### PR DESCRIPTION
**Pull request by cachai2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,20 +167,16 @@ Note: This tutorial requires that the Azure SQL database is setup as shown in [C
 - In the file that opens, replace the 'public static async Task< IActionResult > Run' block with the below code.
 
     ```csharp
-    public static class HttpTriggerCSharp1
+    public static async Task<IActionResult> Run(
```
Comment (cgillum) [inline README.md:170]: Why did you remove the `[FunctionName]` attribute?

Comment (cachai2) [reply README.md:170]: The wording in the readme says to replace the Run block in the code. The [FunctionName] exists outside of this block

Comment (cachai2) [reply README.md:170]: Made a change yesterday to the doc to have a happy landing url for the input binding. When updating the code in the readme block, I copied over a bit too much of the code. This PR is to get it to realign with the wording, "In the file that opens, replace the 'public static async Task< IActionResult > Run' block with the below code."

**Comment by Charles-Gagnon** [comment]
This aligns it with other examples in the file so seems reasonable to update
